README.md

```diff
@@ -55,6 +55,29 @@ test.all(common)
 
 A valid (read: that follows this interface) IPFS core implementation, must expose the following API.
 
+## Files
+
+### `cat` 
+
+> Streams the data contained by an IPFS object(s) at the given IPFS multihash..
+
+##### `Go` **WIP**
+
+##### `JavaScript` - ipfs.cat(multihash, [callback])
+
+`multihash` is a [multihash][] which can be passed as
+
+- Buffer, the raw Buffer of the multihash (or of and encoded version)
+- String, the toString version of the multihash (or of an encoded version)
```

Comment:
I find this wording really unclear -- how can we improve it? Maybe `s/encoded/base58-encoded/`?

Basically what we want to say is: "you can supply a multihash either as a Buffer or a String. Either can be in binary or base58-encoded format."